### ISSUE TITLE
dellemc_idrac_server_config_profile

### ISSUE THREAD

**Issue by alexskrc** [dell/dellemc-openmanage-ansible-modules#57]
Hi there, I have a complex-ish playbook that configures R630 servers for ESXi w/VSAN, and I keep running into issues (the IDRAC AND LC are updated to 2.63.60.61 via ssh in a different task) when I try to setup one-time boot to UEFI PXE. If I load and SCP with the "OneTimeUefiBootSeq" set to "NIC.PxeDevice.1-1" it gets stuck in this condition, and every restart PXE boots, even when "UefiBootSeq" is set to"Disk.SDInternal.1-1".
I changed the workflow a bit and I now load an SCP with "UefiBootSeq" set to PXE, with a shutdown_type: "Forced", and then I load just a short SCP setting the "UefiBootSeq" to"Disk.SDInternal.1-1", with a shutdown_type: "NoReboot", so when the ESXi installer reboots it applies.
The results of this are inconsistent though, it happens a lot that the Import job gets stuck at 90%, even after multiple reboots, so the second task never applies and the server reinstalls ESXi.
I am now adding logic to detect this condition, but is this a known issue? Did anyone else experience this?
thanks!
Alex

**Comment by anupamaloke** [comment]
@alexskrc, if you need to perform a one-time boot to PXE, then maybe you can use the ```redfish_command``` module to do that. See https://github.com/dell/redfish-ansible-module/blob/master/playbooks/bios/set_onetime_boot.yml

**Comment by alexskrc** [comment]
I ended up calling Dell support, I was able to reproduce this behavior using ssh and racadm.
Not sure if there will be a solution for this, we figured out a workaround though:
1. load the SCP setting the boot order to PXE, then SD card; force reboot after load so it applies right away;
2. right after the completion run the following to disable PXE on the next reboot, which will happen after the ESXI installer completed:

- name: Disable PXE
  delegate_to: localhost
  command: "sshpass -p {{ansible_password}} ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no {{ansible_user}}@{{ansible_host}} racadm set BIOS.NetworkSettings.PxeDev1EnDis Disabled"
  async: 300
  poll: 10
  register: bootdeviceset
  until: ("Success" in bootdeviceset.stdout)
  retries: 5
- name: Schedule a job to disable PXE 
  delegate_to: localhost
  command: "sshpass -p {{ansible_password}} ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no {{ansible_user}}@{{ansible_host}} racadm jobqueue create BIOS.Setup.1-1"
  async: 300
  poll: 10
  register: bootdeviceset
  until: ("Success" in bootdeviceset.stdout)
  retries: 5
